lms/static/scripts/frontend_apps/components/LaunchErrorDialog.js

```diff
@@ -219,6 +221,26 @@ export default function LaunchErrorDialog({
         </ErrorModal>
       );
 
+    case 'd2l_student_not_in_group':
+      return (
+        <ErrorModal
+          busy={busy}
+          error={error}
+          title="You're not in any of this assignment's groups"
+        >
+          <p>
+            Hypothesis couldn&apos;t launch this assignment because you
+            aren&apos;t in any of the groups in the assignment&apos;s group set.
+          </p>
+          <p>
+            <b>
+              To fix the problem, an instructor needs to add your D2L user
```

Comment:
I reckon the rationale to special case these is having `D2L` (and Canvas and Blackboard) in the error message.